<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigUI</class>
 <widget class="QMainWindow" name="ConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 42, 82);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,10,0">
      <item>
       <widget class="QPushButton" name="pushButton_load">
        <property name="text">
         <string>加载配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_config">
        <item>
         <property name="text">
          <string>上一次应用配置</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>默认配置</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>删除配置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>曲线1：      </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_line1">
            <item>
             <property name="text">
              <string>不显示</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1-B2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>a（加速度)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1z</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2z</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>曲线2：      </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_line2">
            <item>
             <property name="text">
              <string>不显示</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1-B2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>a（加速度)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1z</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2z</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>预警判据：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_warn">
            <item>
             <property name="text">
              <string>B1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B1-B2</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M31"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>探头1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M32"/>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M33">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1x"/>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1y"/>
            </item>
            <item row="4" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1z"/>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M22">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M23"/>
            </item>
            <item row="2" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M21"/>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M13"/>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M12"/>
            </item>
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_1_M11">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_matrix1">
              <property name="text">
               <string>启动校准</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="3" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M31"/>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M32"/>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M33">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2x"/>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2y"/>
            </item>
            <item row="4" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2z"/>
            </item>
            <item row="2" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M21"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>探头2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M13"/>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M12"/>
            </item>
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M11">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M23"/>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2_M22">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_matrix2">
              <property name="text">
               <string>启动校准</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M22">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M11">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M23"/>
            </item>
            <item row="2" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M21"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>分量一致性</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M13"/>
            </item>
            <item row="3" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M31"/>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M32"/>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M33">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3_M12"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_matrix3">
              <property name="text">
               <string>启动校准</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>其他参数</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>DT:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_DT">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>ML:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_ML">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,0">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>MT:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_MT">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>THm:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_THm">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>THa: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_THa">
            <property name="value">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>采样率:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_sampleRate">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>MGmax:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_MGmax">
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>保存配置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_apply">
          <property name="text">
           <string>应用配置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
   <zorder></zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
